source/dotnet/Library/AdaptiveCards/Rendering/AdaptiveHostConfig.cs

```diff
@@ -43,6 +43,12 @@ public class AdaptiveHostConfig : AdaptiveConfigBase
         /// </summary>
         public bool SupportsInteractivity { get; set; } = true;
 
+        /// <summary>
+        /// Image Base URL for relative URLs
+        /// </summary>
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public string ImageBaseUrl { get; set; } = null;
+
```

Comment:
Should this just be a URI instead of a string. Makes the validation happen earlier.